specification/web/resource-manager/Microsoft.Web/stable/2022-09-01/StaticSites.json

```diff
@@ -3134,6 +3134,251 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/staticSites/{name}/trafficSplitting": {
```

Comment:
Need to be a new api-version

Reply:
I'm not sure what you mean, this is being added in a new api version.